app/models/article.js

```diff
@@ -1,63 +1,35 @@
 import DS from 'ember-data';
+import { inject as service } from '@ember/service';
 
 const { attr, belongsTo, hasMany } = DS;
 
 export default DS.Model.extend({
-  /**
```

Comment:
Removed comments as they don't appear to give much value. Also I don't set a default `date` as those values get set by the server.